srcs/cgi/CGIContext.cpp

```diff
@@ -0,0 +1,99 @@
+# include <cgi/CGIContext.hpp>
+
+namespace cgi
+{
+
+void strcpy(char* me, const char* other){
+    if (!other || !me) { LOG_F("Failed copy"); return; }
+
+    int i;
+    for (i = 0; other[i] != '\0'; i++) {
+        me[i] = other[i];
+    }
+    me[i] = '\0';
+}
+
+CGIContext::CGIContext(std::vector<std::string> queryValues, std::string scriptPath){
```

Comment:
This is all correct, but depending if the request is a GET or a POST, the envp may change. I'll handle it.